src/picongpu/include/simulation_defines/param/flylite.param

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2017 Axel Huebl
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Configuration file for the atomic particle population kinetics model FLYlite.
+ * Its main purpose is non-LTE collisional-radiative modeling for transient
```

Comment:
yes, bit ugly but will keep it in the extended description below since it's a know term in plasma physics (and will still be introduced below)